docker-compose.yml

```diff
@@ -239,6 +239,11 @@ services:
       - './secrets/mirror-intel.env'
```

Comment:
Not directly related to this change, but I would recommend putting some mem usage limits here

Reply:
The memory limit is leveraged because of initial pull requires lots of memory. It will be added back when we are ready to serve the new server.